addons/l10n_it_edi_withholding/models/account_edi_format.py

```diff
@@ -99,15 +101,15 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info):
             if data_kind != 'cassa-prev' or ('enasarco' not in data_text and not 'tc07' in data_text):
                 continue
             enasarco_amount = float(number_text)
-            enasarco_percentage = -self.env.company.currency_id.round(enasarco_amount / price_subtotal * 100)
+            enasarco_percentage = -currency.round(enasarco_amount / price_subtotal * 100)
```

Comment:
Now that we have set a default currency when not specified in the file, we can use it